CONTRIBUTING.md

```diff
@@ -269,6 +269,6 @@ To ensure all your commits are signed, you may choose to add this alias to your
 
 Or you may configure your IDE, for example, Visual Studio Code to automatically sign-off commits for you:
 
-<a href="./.github/assets/images/git-signoff-vscode.png" ><img src="./.github/assets/images/git-signoff-vscode.png" width="50%"/><a>
```

Comment:
I am not sure if the link is broken?

Reply:
> I am not sure if this link is broken?

I found several time that the pic is not showing up. So I thought of refactoring it! 
Let me know if it doesn't looks good.

Reply:
It rendered  perfectly every time for me, btw how do you checked that it is working now? @helper-uttam

Reply:
While editing I checked from preview tab, it was working.

Reply:
i am not sure , i think the previous link is correct 🤔🤔

Reply:
Okay will update it tomorrow! 
Thanks for reviewing.

Reply:
wait, let @Abhijay007 review it, then change the link to the previous one (i may be wrong)

Reply:
it is working fine for me too, I think no need to change that link, kindly revert this change !!